x-pack/platform/plugins/shared/lens/public/app_plugin/shared/edit_on_the_fly/flyout_wrapper.tsx

```diff
@@ -116,6 +116,7 @@ export const FlyoutWrapper = ({
           margin-left: -${euiThemeVars.euiFormMaxWidth};
           pointer-events: none;
           .euiFlyoutBody__overflow {
+            transform: initial;
```

Comment:
Shouldn't we add this also to the data_visualizer field_stats file?
I don't understand why this was copied and pasted there, but it should be better to apply the same style changes

Reply:
No, that's a special case we need to properly display extra drop targets for drag and drop in the dashboard. Doesn't happen for the field_stats.